### PR TITLE
Remove xvfb for saucelabs. Use wct instead of polymer-cli

### PR DESCRIPTION
**Pull request by manolo**
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-element-skeleton/77)

### CODE REVIEW

**Comment by web-padawan** [comment]
:lgtm:

---

Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-element-skeleton/77#-:-KzY9qQE---LXmUDNhE1:bnfp4nl)*

File: .travis.yml [hunk 1]
```diff
@@ -31,11 +31,11 @@ script:
         gemini test test/visual;
       fi &&
       if [[ "$TEST_SUITE" = "unit_tests" ]]; then
-        xvfb-run -s '-screen 0 1024x768x24' polymer test --env saucelabs;
+        wct --env saucelabs;
```
Comment (limonte) [inline .travis.yml]: could you please apply the same change to date-picker to verify that viewport size doesn't matter for tests?

**Comment by manolo** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[.travis.yml, line 34 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-element-skeleton/77#-KzY7QDwLUUo7sQXMhTU-r1-34:-Kzc0AVz2kCHfl198P8d:b5esf7i) ([raw file](https://github.com/vaadin/vaadin-element-skeleton/blob/37d00316fa3a70ca4aa8cb62a3d21e0eb13aa457/.travis.yml#L34)):*
<details><summary><i>Previously, limonte (Limon Monte) wrote…</i></summary><blockquote>

could you please apply the same change to date-picker to verify that viewport size doesn't matter for tests?
</blockquote></details>

applied the same changes to your branch in date-picker and everything works as expected https://travis-ci.org/vaadin/vaadin-date-picker/builds/306198342 . At the same time I opened the PR for date-picker

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-element-skeleton/77)*

**Comment by limonte** [comment]
@web-padawan please apply the same change to all components when you have time.

**Comment by web-padawan** [comment]
@limonte will do.

**Pull request merged**